tests/react/basic.test.tsx

```diff
@@ -34,6 +36,28 @@ const useCommitCount = () => {
   return commitCountRef.current
 }
 
+/* eslint-disable testing-library/no-unnecessary-act */
+const renderRoot = async (element: ReactNode) => {
```

Comment:
This is the only hack I came up with. If it looks good, I'll work on other files too.

Reply:
It seems like we should await all `act()`. `render` from RTL throws away the promise from `act()` call.
https://github.com/testing-library/react-testing-library/blob/main/src/pure.js#L145

This workaround isn't ideal because we can't test the initial "Loading...".